src/main/webapp/utilities/MapsInterface.js

```diff
@@ -0,0 +1,105 @@
+class MapInterface {
+    /**
+     * @param element the DOM element the map to be rendered
+     * @param centerCoordinates coordinates object that will be the center of the map
+     * @param options (optional) additional parameters that might be needed
+     */
+    constructor(element, centerCoordinates, options = {}) {
```

Comment:
Can we have a default options object rather than all this options.X || some_default_value?

Reply:
Done! Can't do this on `addMarker`, because the default option depends on another coordinate, so I am keeping as it is there.